content/en/news/releases/1.13.x/announcing-1.13.9/index.md

```diff
@@ -7,7 +7,12 @@ publishdate: 2022-10-11
 release: 1.13.9
 ---
 
-This release contains a patch for [CVE-2022-41715](https://github.com/golang/go/issues/55949) and bug fixes to improve robustness. This release note describes what is different between Istio 1.13.8 and Istio 1.13.9.
+This release contains a fix for [CVE-2022-39278](/news/security/istio-security-2022-007/#cve-2022-39278) and
+bug fixes to improve robustness. We recommend users install this release instead of Istio 1.13.8,
+which does not contain the above CVE fix.
+This release note describes what is different between Istio 1.13.8 and Istio 1.13.9.
+
+This release also includes fixes to improve robustness.
```

Comment:
"This release also includes fixes to improve robustness." can be removed because it is mentioned in the line above.